apps/ngx-env-demo/package.json

```diff
@@ -1,54 +1,47 @@
 {
-  "name": "@ngx-env/demo",
-  "version": "16.1.1",
-  "private": true,
+  "name": "ngx-env-demo",
+  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "cross-env NODE_ENV=development ng serve",
-    "prepare:build": "npx ng config projects.ngx-env-demo.architect.build.options.ngxEnv.prefix 'NGX_'",
-    "build": "ng build",
-    "prepare:test": "npx ng config projects.ngx-env-demo.architect.test.options.ngxEnv.prefix 'NGX_'",
-    "test": "cross-env NODE_ENV=test NGX_BRANCH_NAME=main ng test --watch=false",
-    "prepare:server": "npx ng config projects.ngx-env-demo.architect.server.options.ngxEnv.prefix 'NGX_'",
-    "dev:ssr": "ng run ngx-env-demo:serve-ssr",
-    "serve:ssr": "node dist/ngx-env-demo/server/main.js",
-    "build:ssr": "ng build && ng run ngx-env-demo:server",
-    "prerender": "ng run ngx-env-demo:prerender"
+    "watch": "ng build --watch --configuration development",
+    "start": "NODE_ENV=development ng serve",
+    "prepare:build": "npx ng config projects.ngx-env-demo.architect.build.options.ngxEnv.prefix 'NG_APP_'",
```

Comment:
NG_APP is actually the default prefix so if we want to use it in our code there is no real need to add it ngxEnv conf hence the idea to use NGX is to test that overloading it actually works

Reply:
ok, so I've to restore NGX_?

Reply:
yes please 👌🏼